edit-post/components/header/plugins/index.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import { map, isEmpty, isString } from 'lodash';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withInstanceId, IconButton, MenuItemsGroup } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getMoreMenuItems } from '../../../api/more-menu-item';
+import { getActivePlugin, getOpenedGeneralSidebar } from '../../../store/selectors';
+
+/**
+ * Renders a list of plugins that will activate different UI elements.
+ *
+ * @param   {Object} props The component props.
+ * @return {Object} The rendered list of menu items.
```

Comment:
While it's not explicitly documented as such, the convention here and in the documentation standards examples encourage separating tags via newline:

```js
/**
 *
 * @param {Object} props The component props.
 *
 * @return {Object} The rendered list of menu items.
 */
```

Reply:
More accurately we could describe the return type as an element (`WPElement`, though not formally defined. Maybe it ought to be).